### PR TITLE
PHP8.4 Support Added

### PR DESCRIPTION
**Pull request by cod43156**
|    Q          |   A
|-------------- | ------
| Documentation | yes/no
| Bugfix        | yes/no
| BC Break      | yes/no
| New Feature   | yes/no
| RFC           | yes/no
| QA            | yes/no

### CODE REVIEW

**Comment by gsteel** [comment]
@cod43156 Please rebase onto `3.1.x` the docs linting issues have been fixed. Please also sign off your commits as per the [contributors guide](https://github.com/laminas/.github/blob/main/CONTRIBUTING.md#commit-signoffs)

**Comment by cod43156** [comment]
> @cod43156 Please rebase onto `3.1.x` the docs linting issues have been fixed. Please also sign off your commits as per the [contributors guide](https://github.com/laminas/.github/blob/main/CONTRIBUTING.md#commit-signoffs)

@gsteel Rebase onto 3.1.x is done, and all commits have been signed off as requested.

File: .laminas-ci.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
         "igbinary"
     ],
     "ignore_php_platform_requirements": {
-        "8.3": true
+        "8.3": true,
```
Comment (gsteel) [inline .laminas-ci.json]: Platform requirements no longer need to ignored on 8.3

**Review by gsteel — CHANGES_REQUESTED** [review]

File: .laminas-ci.json (hunk 1 above)
Comment (cod43156) [reply .laminas-ci.json]: This change has been implemented

**Review by gsteel — APPROVED** [review]
Thank you @cod43156

**Pull request merged**